roles/default/kiali-deploy/defaults/main.yml

```
@@ -58,6 +58,9 @@ kiali_defaults:
     image_pull_secrets: []
     image_version: ""
     ingress_enabled: true
+    logger:
+      log_level: info
+      log_format: text
```

Even though this is valid, for consistency with the rest of the values in this file, please wrap string values in quotes:
```suggestion
      log_level: "info"
      log_format: "text"
```